docs/partial_source/deep_dive/2_backend_setting.rst

```diff
@@ -62,6 +62,34 @@ backend is unset using the :code:`ivy.unset_backend`, the `implicit_backend`_ wi
 assume the backend from the last run. While the `implicit_backend`_ functionality gives more freedom to the user , the \
 recommended way of doing things would be set the backend explicitly.
 
+
+Backend and Frontend Version Support
```

Comment:
this should be a sub-heading, with `-` rather than `=` for the underline.

Reply:
done